README.md

```diff
@@ -1,5 +1,148 @@
+[![GitHub Last Commit](https://img.shields.io/github/last-commit/xlibb/module-asb.svg)](https://github.com/xlibb/module-asb/commits/master)
+
+[Azure Service Bus](https://azure.microsoft.com/en-us/services/service-bus/) is a robust cloud-based 
+Messaging as a Service (MaaS) offering. It serves as an enterprise message broker, providing both message queues and 
+publish-subscribe topics. The service bus plays a vital role in decoupling applications and services from one another. 
+Messages are utilized for seamless data transfer between different applications and services.
+
+The Ballerina Admin client connector for Azure Service Bus facilitates connectivity to Azure Service Bus, 
+enabling the creation and deletion of Azure Service Bus topics and subscriptions.
+
+### Admin client
+
+Azure service bus Admin client provides administrative capabilities to create and delete Azure service bus topics
+and subscriptions. In addition to the creation and deletion functionalities for topics and subscriptions, this module
+also provides the ability to check the existence of an Azure Service Bus topic or subscription.
+
+#### Create admin client
+
+An `asb.admin:Client` can be created as follows.
+```ballerina
+import xlibb/asb.admin as admin;
+
+configurable string connectionString = ?;
+
+final admin:Client clientEp = check new(connectionString);
+```
+
+Following APIs are available in the Azure service bus admin client:
+- <b> topicExists </b>: Checks whether an ASB `topic` is available in the service bus resource.
+- <b> createTopic </b>: Creates an ASB `topic` in the service bus resource.
+- <b> deleteTopic </b>: Deletes an ASB `topic` from the service bus resource.
+- <b> subscriptionExists </b>: Checks whether an ASB `subscription` exists in the service bus resource.
+- <b> createSubscription </b>: Creates an ASB `subscription` in the service bus resource.
+- <b> deleteSubscription </b>: Deletes an ASB `subscription` from the service bus resource.
+
+##### Topic exists
+
+This method checks whether the given topic is available in the service bus namespace. User has to provide the name of
+the topic as an input to the function.
+```ballerina
+import xlibb/asb.admin as admin;
+
+configurable string connectionString = ?;
+
+public function main() returns error? {
+    admin:Client clientEp = check new(connectionString);
+    boolean topicExists = check clientEp->topicExists("topic-a");
+}
+```
+
+In the event of a network failure during execution, it will lead to an `asb.admin:Error`.
+
+##### Create topic
+
+This method creates a new topic in the Azure service bus namespace. User has to provide the name of the topic as an
+input to the function.
+```ballerina
+import xlibb/asb.admin as admin;
+
+configurable string connectionString = ?;
+
+public function main() returns error? {
+    admin:Client clientEp = check new(connectionString);
```

Comment:
do we need to always showcase how the client is created?
cant we simple add,
```bal
admin:TopicCreated topicCreated = check clientEp->createTopic("topic-a");
```

Reply:
Fixed with [4725ee04a0664cc0fdc89379b429f721257297b2](https://github.com/xlibb/module-asb/pull/14/commits/4725ee04a0664cc0fdc89379b429f721257297b2)